pkg/varlinkapi/images.go

```diff
@@ -382,11 +382,11 @@ func (i *LibpodAPI) RemoveImage(call ioprojectatomicpodman.VarlinkCall, name str
 	if err != nil {
 		return call.ReplyImageNotFound(name)
 	}
-	imageID, err := runtime.RemoveImage(ctx, newImage, force)
+	_, err := runtime.RemoveImage(ctx, newImage, force)
 	if err != nil {
 		return call.ReplyErrorOccurred(err.Error())
 	}
-	return call.ReplyRemoveImage(imageID)
+	return call.ReplyRemoveImage(newImage.ID())
```

Comment:
does the value of newImage.ID() still exist after the removal at line 385?  Is that the reason for the cranky tests?

Reply:
yeah the ID is still known ... the errors are a dumb mistake apparently.